frontend/src/components/ResourceInfo.tsx

```diff
@@ -102,9 +105,11 @@ export class ResourceInfo extends React.Component<ResourceInfoProps, {}> {
         <h2 className={commonCss.header2}>Custom Properties</h2>
         <dl className={css.resourceInfo}>
           {customPropertyMap.getEntryList().map(k => (
-            <div className={css.field} key={k[0]}>
-              <dt className={css.term}>{k[0]}</dt>
-              <dd className={css.value}>
+            <div className={css.field} key={k[0]} data-testid='resource-info-property'>
+              <dt className={css.term} data-testid='resource-info-property-key'>
```

Comment:
Good points, let me explain some reasons.

1. We do not need to distinguish property and custom property going forward, because we will only use custom properties

2. Testing only expected custom properties are rendered is by design. There are different testing strategies, if we further want to ensure their locations in the page, we can write a full page snapshot, but a full page snapshot is also long and unfocused. I intentionally only test that the properties are on the screen to make test intention more explicit and not include implementation details that I didn't want to test. E.g. when we revamp this page the new test might not even need a change.